pom.xml

```diff
@@ -432,6 +432,35 @@
       </properties>
     </profile>
 
+    <!-- Copy newly built native libraries back to source tree -->
+    <profile>
+      <id>preserve-natives</id>
```

Comment:
I feel that "preserve" is a misleading word here.  In my opinion, this would normally suggest to keep the OLD copy.

Reply:
How `update-precompiled-resources`?